hometask 4.py

```diff
@@ -0,0 +1,33 @@
+name = input('Enter a name:')
+
+number = input('Enter a number:')
+
+
+phonebook = {name:number}
+
+while True:
+    user_input = input('Enter a command:')
+    split_input = user_input.split()
+
+    command = split_input[0]
+
+    if command == 'stats':
+        print(len(phonebook))
+    elif command == 'add':
+         for i in range(1):
+             name=input('Enter a name:')
+             number=input('Enter a number:')
+             phonebook[name]=number
```

Comment:
There shouldn't exist an ability to override records. Please, make additional check here

Reply:
There is no such lines in code anymore. Please revise.